pkgs/applications/science/robotics/inav-configurator/default.nix

```diff
@@ -0,0 +1,54 @@
+{lib, stdenv, fetchurl, makeDesktopItem, nwjs, wrapGAppsHook, gsettings-desktop-schemas, gtk3 }:
+
+let
```

Comment:
The let is not needed because the attribute set is a rec which means attributes can reference each other.